app/login.js

```diff
@@ -0,0 +1,46 @@
+/* var users=JSON.parse(usersJSON);
+
+function clicked(){
+
+
+for (i = 0 ; i < users.length ; i++) {
+
+  var  email = document.getElementById('email').value;
+  var  password = document.getElementById('password').value;
+
+  if (email == users[i].email) {
+
+      window.alert("Welcome " + users[i].userName ); break;
+
+    } else {
+
+      window.alert("You are not registered. Need to Sign Up !"); break;
+    }
+
+  }
+
+} */
+
+function checkUser () {
+
+  var email = document.getElementById( 'email' ).value;
+  var password = document.getElementById( 'password' ).value;
+
+  if ( accountService.checkLogin (email , password) == "1" ) {
```

Comment:
Just call the checkLogin function once and assign the result to a properly named variable( hint loginResultCode ). Use a switch statement to check the code